src/adapter/src/coord.rs

```diff
@@ -3477,12 +3477,29 @@ impl Coordinator {
     }
 
     /// Publishes a notice message to all sessions.
-    pub(crate) fn broadcast_notice(&self, notice: AdapterNotice) {
+    pub fn broadcast_notice(&self, notice: AdapterNotice) {
```

Comment:
I made this pub so it wouldn't emit an unused code warning. It felt like a good API to keep around, happy to just remove it if folks think that's better